BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -45,6 +63,9 @@ public struct AccessibilityElement: Element {
     /// See [Accessibility Activate Documentation](https://developer.apple.com/documentation/objectivec/nsobject/1615165-accessibilityactivate) for further information.
     public var accessibilityActivate: (() -> Bool)? = nil
 
+    /// The array contains one or more `Custom Action`s, defining additional supported actions. Assistive technologies, such as VoiceOver, will display your custom actions to the user at appropriate times.
+    public var customActions: [CustomAction]? = nil
```

Comment:
We need to add this to the `public func accessibilityElement(` below.

Reply:
I was imagining that this would be such a rare occurrence you'd be fine using the configure block in the  big initializer, but if you don't think adding yet another property to the convenience method is overkill I've got no pushback.

Reply:
I think adding it and giving it an optional `= nil` is fine, but we tend to avoid folks using the `init` based builders to avoid the "pyramid of doom" of intending. Another option is to throw a `configure` block on the `.accessibilityElement` builder.

Reply:
+1 to adding this to the `public func` modifier.